modules/features/filters/src/main/java/au/com/shiftyjelly/pocketcasts/playlists/create/SmartPlaylistPreviewPage.kt

```diff
@@ -548,7 +548,41 @@ private fun AppliedRules.description(ruleType: RuleType) = when (ruleType) {
         null -> null
     }
 
-    RuleType.EpisodeStatus -> TODO()
+    RuleType.EpisodeStatus -> {
+        if (episodeStatus != null) {
+            when {
+                episodeStatus.unplayed -> when {
+                    episodeStatus.inProgress && episodeStatus.completed -> {
```

Comment:
I believe this is the same as not using the episode status rule? In the current Filters create page, it would not be toggled on if this were the user's option. I wonder if this means the rule shouldn't be listed.